webapp/javascript/redux/reducers/settings.ts

```diff
@@ -231,20 +265,44 @@ export const deleteAPIKey = createAsyncThunk(
   }
 );
 
+export const deleteApp = createAsyncThunk(
+  'newRoot/deleteApp',
+  async (app: App, thunkAPI) => {
+    const res = await deleteAppAPI({ name: app.name });
+
+    // eslint-disable-next-line @typescript-eslint/no-floating-promises
```

Comment:
That's true we do have this rule, but I wonder why it doesn't trigger in other places like here https://github.com/pyroscope-io/pyroscope/blob/main/webapp/javascript/redux/reducers/continuous.ts#L230

Reply:
it does, but warnings only